### DISCUSSION TITLE
Trapezoid form in CSS

### DISCUSSION THREAD

**Discussion by mihaimazareanu** [neuefische/questions#437] [Web] [answered]
Hi,

I am trying to draw a trapezoid form with Styled Components, using borders. 

My code is like this:

    const StyledContainer = styled.div`
    position: relative;
    border-right: 75px solid red;
    border-top: 25px solid transparent;
    height: 50%;
    width: 75%;
    display: flex;
    
However, for my purposes, I need the border-right and border-top properties to be responsive. But when I insert 75% instead of 75px in `border-right`, the trapezoid is not visible anymore.

I've tried the whole day to fix this but I didn't manage. Anyone have any idea what I need to do? 🙏

**Comment by actyralf** [comment, marked as answer]
`border-width` does not support percentages [see documentation](https://stackoverflow.com/questions/13474754/how-to-set-borders-thickness-in-percentages).
Here is a workaround: https://stackoverflow.com/questions/13474754/how-to-set-borders-thickness-in-percentages (I didn't test this though)
